includes/class-newspack-newsletters-logger.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Newspack Newsletters Logger
+ *
+ * @package Newspack
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Logger.
+ */
+class Newspack_Newsletters_Logger {
+	/**
+	 * A logger.
+	 *
+	 * @param any $payload The payload to log.
+	 */
+	public static function log( $payload ) {
+		if ( ! defined( 'NEWSPACK_LOG_LEVEL' ) || 0 > (int) NEWSPACK_LOG_LEVEL ) {
+			return;
+		}
+
+		$header = 'NEWSPACK-NEWSLETTERS';
+		if ( class_exists( '\Newspack\Logger' ) ) {
+			\Newspack\Logger::log( $payload, $header );
+		} else {
+			error_log( '[' . $header . ']: ' . $message ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
```

Comment:
`$message` is not defined

Reply:
Fixed in 4ec4724835f7394d2e69b9e011021971704f3090